commands/cat_end.py

```diff
@@ -0,0 +1,49 @@
+from db.dao.sector_dao import SectorDao
+from base.command import BaseCommand
+from base.arguments import StringArgument
+from db.base import sector_size
+from db.dao.file_dao import FileDao
+
+
+class CatEndCommand(BaseCommand):
+    command = 'cat_end'
+    arguments = [StringArgument()]
+
+    def run(self):
+        filename = self.arguments[0].data
+        current_directory = self.context.current_directory
+
+        if (FileDao.is_unique_filename(filename, current_directory)):
+            self.log('InvalidEntry: File does not exist', prefix=False)
+            # raise ValueError('File does not exist')
```

Comment:
Raising ValueError is okay, might be better, if we want to modify that all errors should be shown in a red color.

Reply:
wo program band hojata :')

Reply:
Ah, the behavior is fixed in #23. The program will not end after that is merged, so we keep raising errors.